spec/run_results_combiner_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require "helper"
+
+if SimpleCov.usable?
+  describe SimpleCov::RunResultsCombiner do
+    describe "with two faked coverage resultsets" do
+      before do
+        @resultset1 = {
```

Comment:
Instead of using `before` and `@resultset` I think `let(:resultset1)` etc. should do the job - what do you think? :)